test/suite/test_tiered04.py

```diff
@@ -47,7 +47,8 @@ class test_tiered04(wttest.WiredTigerTestCase):
             bucket = get_bucket1_name('s3_store'),
             bucket1 = get_bucket2_name('s3_store'),
             prefix = generate_s3_prefix(),
-            prefix1 = generate_s3_prefix(),
+            # Test that object name with "/" are processed. 
+            prefix1 = generate_s3_prefix() + "/s3/source/",
```

Comment:
We could probably extend this test further to test multiple directories but we could add that to another ticket.

Reply:
Hi Sean, if you have a look at the changes in [helper_tiered.py](https://github.com/wiredtiger/wiredtiger/pull/7792/files#diff-3b9a97cd9c6c6d969871e2d6bca51c6db3de3cdf63e84e3ba842ed54d6eb0f37) I have changed the prefix generation so that they all include "/" and this will test multiple directories with the changes.

Reply:
Ah okay, thanks for the clarification! I misunderstood helper_tiered.py.